portfolio/src/main/webapp/script.js

```diff
@@ -98,8 +98,18 @@ window.addEventListener('scroll', function() {
  * and adds it to the DOM.
  */
 function getComment() {
-  let url = '/data?comment-limit=' + document.getElementById("quantity").value.toString();
-  fetch(url).then(response => response.json()).then((comments) => {
+  let dataUrl = '/data?comment-limit=';
+  let limitNumber = 0;
+  try {
+    limitNumber = document.getElementById("quantity").value;
+    if(limitNumber < 0) throw 'Invalid number (cannot be negative): '
+        + limitNumber.toString();
+  } catch(err) {
+    console.log(err);
+    limitNumber = 0;
```

Comment:
Same here, can you add a comment to the function-level comment, or before / in this try/catch block clarifying that you fallback to 0 if a negative number is somehow passed here?

Reply:
Done!